### PR TITLE
Minimally tested sb*iw support.

### PR DESCRIPTION
**Pull request by jim-wilson**

### CODE REVIEW

**Comment by aswaterman** [comment]
Thanks for quickly putting this together, @jim-wilson!

**Review by kito-cheng — APPROVED** [review]
Verified with following testcase:

```
int foo1(int x, int y){
    return (x + y)| 0x8000;
}
int foo2(int x, int y){
    return (x + y) ^ 0x8000;
}
int foo3(int x, int y){
    return (x + y) & (~0x8000);
}
```

File: gcc/config/riscv/bitmanip.md [hunk 1]
```diff
@@ -153,6 +153,14 @@
   "sbsetw\t%0,%1,%2"
   [(set_attr "type" "bitmanip")])
 
+(define_insn "*sbsetiw"
+  [(set (match_operand:DI 0 "register_operand" "=r")
+	(ior:DI (sign_extend:DI (match_operand:SI 1 "register_operand" "=r"))
```
Comment (kito-cheng) [inline gcc/config/riscv/bitmanip.md:158]: Redundant `=` for input operand, which is reported by Evgeny before: https://github.com/riscv/riscv-bitmanip/issues/79

**Comment by kito-cheng** [comment]
I also created another branch based on gcc 10.2 https://github.com/riscv/riscv-gcc/tree/riscv-gcc-10.2.0-rvb , and I will run test with that branch :)

**Comment by jim-wilson** [comment]
I fixed the =r input operatnd problems in my patch and in the surrounding patterns.

**Comment by jim-wilson** [comment]
Sorry, I didn't read that clearly enough.  I didn't realize you already fixed the =r problem on the new branch.  I can move to the new branch and submit a pr there.

**Pull request merged**